CMakeLists.txt

```diff
@@ -364,6 +364,7 @@ endif()
 list(APPEND SUNSHINE_DEFINITIONS SUNSHINE_ASSETS_DIR="${SUNSHINE_ASSETS_DIR}")
 list(APPEND SUNSHINE_DEFINITIONS SUNSHINE_CONFIG_DIR="${SUNSHINE_CONFIG_DIR}")
 list(APPEND SUNSHINE_DEFINITIONS SUNSHINE_DEFAULT_DIR="${SUNSHINE_DEFAULT_DIR}")
+configure_file("${CMAKE_CURRENT_SOURCE_DIR}/NOTICE" "NOTICE")
```

Comment:
One last change, undo adding this line. We are going to add the notice to the web UI (separate PR for that). After this change I will merge this PR. Thanks!

Reply:
Done